src/kubeseal_auto/secrets/prompts.py

```
@@ -0,0 +1,218 @@
+    if not name:
+        return "Name cannot be empty"
+    if len(name) > _DNS_SUBDOMAIN_MAX_LENGTH:
+        return f"Name must be {_DNS_SUBDOMAIN_MAX_LENGTH} characters or less"
+    if not re.match(_DNS_SUBDOMAIN_PATTERN, name):
+        return "Name must consist of lowercase alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character"
+    return True
+
+
+def collect_secret_parameters(
+    namespaces: list[str],
+    detached_mode: bool = False,
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)